src/resources/UsageAnalytics/Read/Exports/ExportsInterfaces.ts

```diff
@@ -87,10 +87,88 @@ export interface GenerateExportParams extends EstimateExportParams {
+     */
+    from: string;
+    /**
+     * The end date of the date range. ISO8601 format 'YYYY-MM-DDThh:mm:ss.sssZ'
+     */
+    to: string;
+    /**
+     * A file name for the export. Can only contain US-ASCII characters. If none is supplied it will be set to the export's id.
+     */
+    filename?: string;
+    /**
+     * Optional description for the export.
+     */
+    description?: string;
+    /**
+     * The dimensions that will be exported. If not provided, all dimensions will be exported.
+     */
+    dimensions?: string[];
+    /**
+     * The format of the generated CSV files. If not provided, it will default to Excel.
+     */
+    exportFormat?: CSVFileFormat;
+    /**
+     * Whether to use the display names in the export's header. If false, the api names will be used. If not provided, it defaults to false.
+     */
+    useDisplayNames?: boolean;
+}
+
+export interface GenerateExportWithBodyParams extends BaseGenerateExportWithBodyParams {
+    /**
+     * The filter that will be applied to the events common dimensions (shared by all types of events).
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    commonFilters?: string[];
+    /**
+     * The filter that will be applied to the click and search events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    searchesFilters?: string[];
+    /**
+     * The filter that will be applied to the custom events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    customEventsFilters?: string[];
+    /**
+     * The tables that will be exported. If not provided, all tables will be exported.
+     */
+    tables?: ExportTablesType[];
+}
+
+export interface GenerateVisitExportWithBodyParams extends BaseGenerateExportWithBodyParams {
+    /**
+     * The filter that will be applied to the events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    inclusionFilters?: string[];
+    /**
+     * Each specified filter is inverted in order to hide events.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    hideEventFilters?: string[];
+    /**
+     * The filter that will be applied to dimensions to exclude events from the results.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    exclusionFilters?: string[];
+    /**
+     * Whether to order the rows by datetime in the export.
+     * If the number of rows exported is to great, this parameter will be ignored and data will be unsorted.
+     * If not provided, it defaults to true.
+     */
+    ordered?: boolean;
+}
+
 export interface GenerateVisitExportParams extends GenerateExportParams {
     /**
      * Whether to order the rows by datetime in the export.
      * If the number of rows exported is too large, this parameter will be ignored and data will be unsorted.
```

Comment:
If there is a large number of rows to export, this parameter will be ignored and the data will be unsorted.